code/src/component/GameStartContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { game } from '../reducers/game';
+import { Button } from './Button';
+import { StartGameScreen, StartGameCard, StartText, Input } from '../styling/startscreen';
+
+export const GameStartContainer = () => {
+    const dispatch = useDispatch();
+    const [inputValue, setInputValue] = useState('');
+
+    // Update store with given username
+    const setUserName = () => {
+        dispatch(game.actions.updateUserName(inputValue));
```

Comment:
Nothing wrong with 
`dispatch(game.actions.setUsername(inputValue));`

if you were inclined to name the action like this